instrumentation/dalli/lib/opentelemetry/instrumentation/dalli/utils.rb

```diff
@@ -53,7 +53,17 @@ def opname(operation, multi)
 
       def format_command(operation, args)
         placeholder = "#{operation} BLOB (OMITTED)"
-        command = [operation, *args].join(' ').strip
+
+        command = +operation.to_s
+        args.flatten.each do |arg|
+          str = arg.to_s
+          if str.bytesize >= CMD_MAX_LEN
+            command << ' BLOB (OMITTED)'
```

Comment:
How about making `'  BLOB (OMITTED)'` a variable `delimiter` which can be shared at line 55 and line 61?

Reply:
It's already shared because we have `# frozen_string_literals: true`, so identical literal strings point to the same object.